gateway_code/tests/rest_server_test.py

```diff
@@ -205,13 +205,6 @@ def test_flash_function(self):
         ret = self.server.post('/open/flash', upload_files=[])
         self.assertEquals(1, ret.json['ret'])
 
-    def test_flash_idle(self):
```

Comment:
This looks like a rebase issue (are maybe no rebase against lastest master) ?

Reply:
OK, it was a rebase issue, I've put it back